### ISSUE TITLE
Memestra doesn't work if you're using __init__ imports

### ISSUE THREAD

**Issue by marimeireles** [QuantStack/memestra#12]
I found a case where memestra doesn't work, but I suppose we want it to work:

If you're importing something from init like in here:

https://github.com/numpy/numpy/blob/41e254f96e8d5bd558d2edbf8b198eb4143e8b74/numpy/__init__.py#L155

for example, memestra won't return anything.

A reproducible example of what I mean would be running a file `init_test.py` with the command `memestra init_test.py --decorator=decoratortest.deprecated` that contains:

```
from tests.misc import foo

foo()
```

in memestra home and adding this line `from some_module import foo` to the `__init__.py` in `tests/misc`.

**Comment by serge-sans-paille** [comment]
This looks like a case where we want the import to be recursive, right? If that's so it would be good to have the generic opt-in recursive import mechanism, then use it for ``__init__.py``

**Comment by marimeireles** [comment]
Yeah, so I thought, but it's not working with the recursion I implemented here https://github.com/QuantStack/memestra/pull/9, that's why I opened a PR to keep track and explore it later. :)
Please let me know if I can assist you with anything regarding these recursion issues.

**Comment by marimeireles** [comment]
Yeah, I think what I meant here was that I was having a problem with relative imports, that are still not working. Maybe we should do this before tagging a release. I can give it a try :)

**Comment by serge-sans-paille** [comment]
Sure, go ahead!

**Comment by marimeireles** [comment]
I think it'd be better to merge your PR first?
Or you don't think it'd make much difference?

**Comment by serge-sans-paille** [comment]
I think it's better to merge my PR first, but it shouldn't prevent you from starting to work on that issue :-)
